iati/iati/templates/500.html

```diff
@@ -1,17 +1,13 @@
-<!DOCTYPE html>
```

Comment:
As the only differences between this and `iati/iati/templates/404.html`, we should make this into a new template, perhaps called `error-base.html`. Then the `500` and `404` pages can set their own `H1` and `p` content.

Reply:
@dalepotter Done.